### ISSUE TITLE
Installing on macOS 10.13.1 APFS

### ISSUE THREAD

**Issue by WardsParadox** [clburlison/pinpoint#29]
Package Installed, but there were errors:
```
Nov 21 15:38:30 TECH-Zack McCauley installd[5400]: PackageKit: Executing script "./postinstall" in /private/tmp/PKInstallSandbox.Jdamg0/Scripts/com.clburlison.pinpoint.QqlKzp
Nov 21 15:38:30 TECH-Zack McCauley installd[5400]: ./postinstall: /Library/LaunchDaemons/com.clburlison.pinpoint.plist: service already loaded
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall: Traceback (most recent call last):
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:   File "/Library/Application Support/pinpoint/bin/pinpoint", line 816, in <module>
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:     main()
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:   File "/Library/Application Support/pinpoint/bin/pinpoint", line 787, in main
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:     ReverseLookup().address_lookup(address_lookup_service)
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:   File "/Library/Application Support/pinpoint/bin/pinpoint", line 663, in address_lookup
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall:     add = dict(Address=str(address))
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: ./postinstall: UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 3: ordinal not in range(128)
Nov 21 15:38:40 TECH-Zack McCauley installd[5400]: PackageKit: Writing receipt for com.clburlison.pinpoint to /
```
Latest release package, pinpoint-2.0.0.83.pkg

**Comment by clburlison** [comment]
Yay Unicode errors. This can be replicated with the following snippet:

```python
        emDash = u'\u2014'
        address = "a{0}b".format(emDash)
        add = dict(Address=address)
```

**Comment by clburlison** [comment]
@WardsParadox Mind trying the following build? You'll want to test in the same location from yesterday since this is a address specific issue. 

[pinpoint-2.0.0.84.pkg.zip](https://github.com/clburlison/pinpoint/files/1496625/pinpoint-2.0.0.84.pkg.zip)

**Comment by WardsParadox** [comment]
Looks good here! Also, thank you for signing your packages.

**Comment by clburlison** [comment]
Fixed with e59ef6d127201f30d68639b2cc09eb87c6028d36
